wurst/network/GamecacheBuffer.wurst

```diff
@@ -1,128 +0,0 @@
-package GamecacheBuffer
-import GamecacheKeys
-import Execute
-import NetworkConfig
-import ErrorHandling
-import SafetyChecks
-
-interface SynchronizerFunction
-	function sync(string key)
-
-public enum BufferType
```

Comment:
Delete instead of deprecate?

Reply:
yes, the gamecache syncing does not work anymore.